src/evict/evict_page.c

```diff
@@ -496,7 +494,7 @@ __evict_child_check(WT_SESSION_IMPL *session, WT_REF *parent)
                                 * If the lookaside history is obsolete, the reference can be
                                 * ignored.
                                 */
-            if (__wt_page_las_active(session, child))
+            if (child->has_las)
```

Comment:
Before durable history, the child page that has las information eviction has a problem, isn't it true any more right? Do we still need this check?

And also, do we really need to it now to represent `ref->state` is `WT_REF_LOOKASIDE`?